cmd/server-main.go

```diff
@@ -1103,18 +1106,12 @@ func serverMain(ctx *cli.Context) {
 		printStartupMessage(getAPIEndpoints(), err)
 
 		// Print a warning at the end of the startup banner so it is more noticeable
-		if newObject.BackendInfo().StandardSCParity == 0 {
-			warnings = append(warnings, color.YellowBold("- The standard parity is set to 0. This can lead to data loss."))
+		if newObject.BackendInfo().StandardSCParity == 0 && !globalIsErasureSD {
```

Comment:
It helps to have this warning about parity displayed for single drive deployments. It serves as a reminder to those who store their data in SNSD setups.

Reply:
It is AB who wanted to avoid it since it's implicit with single drive. We do not have to print something which should be obvious.